pkg/controller/authentication/routes.go

```diff
@@ -328,6 +328,22 @@ func (r *ReconcileAuthentication) reconcileRoute(ctx context.Context, instance *
 			}
 		}
 
+		reqLogger.Info("Found key in observed route", observedRoute.Spec.TLS.Key)
```

Comment:
We shouldn't log out credentials like this by default; if we want to leave these logs in, please use `reqLogger.V(<some number above 0).Info()` instead so that they only show up at higher logging levels.